modules/rules_python_gazelle_plugin/0.22.0/MODULE.bazel

```diff
@@ -0,0 +1,20 @@
+module(
+    name = "rules_python_gazelle_plugin",
+    version = "0.22.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_python", version = "0.18.0")
+bazel_dep(name = "rules_go", version = "0.38.1", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "gazelle", version = "0.31.0", repo_name = "bazel_gazelle")
```

Comment:
The `MODULE.bazel` file in the source archive references `0.29.0` instead, which causes CI to fail.

Reply:
Ah, I looked it up at HEAD, and there it's version 0.31.0. Probably changed very recently. I'll fix it